src/Bootstrap/Types.ts

```diff
@@ -1,9 +1,19 @@
 const TYPES = {
   Logger: Symbol.for('Logger'),
   HTTPClient: Symbol.for('HTTPClient'),
+  ValetKeyGenerator: Symbol.for('ValetKeyGenerator'),
```

Comment:
We can put this one in the `// Services` or `// other` section

Reply:
Actually this can be removed along with the `ValetKeyGenerator` interface. Forgot to do that when I was redesigning. In the current design no abstraction is needed over `generateValetToken` as it was reduced to a simple function that returns a string token. Abstractions may come in later when using these tokens to perform operations on potentially different data stores.

Anyway removed this for now.